tests/drivers/spi/spi_loopback/boards/numaker_pfm_m467.overlay

```diff
@@ -0,0 +1,34 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+
+&pinctrl {
+/* CS/CLK/MISO/MOSI --> EVB's NU5: SS/CLK/MISO/MOSI --> PA11/PA10/PA9/PA8 */
```

Comment:
Do you really need the comment? The pin names have that information already, this is one more thing to get stale if you end up copypasting it on new boards. I'd suggest dropping it.

Reply:
Could I only keep `/* EVB's NU5: SS/CLK/MISO/MOSI */` ?
else user can't easy find the pad "NU5" to flywire MISO/MOSI for SPI loopback test.

Reply:
Oh yeah that makes sense.